Example/src/LightboxExample.tsx

```diff
@@ -34,25 +35,36 @@ const NUMBER_OF_IMAGES = 4;
 const IMAGE_SIZE =
   (dimensions.width - GUTTER_WIDTH * (NUMBER_OF_IMAGES - 1)) / NUMBER_OF_IMAGES;
 
-const styles = StyleSheet.create({
-  container: {
-    paddingTop: 0,
-    height:
-      Platform.OS === 'web' ? dimensions.height - Header.HEIGHT : undefined,
-  },
-
-  scrollContainer: {
-    flexDirection: 'row',
-    flexWrap: 'wrap',
-  },
-
-  backdrop: {
-    ...StyleSheet.absoluteFillObject,
-    backgroundColor: 'black',
-  },
-});
+type ExampleImage = {
```

Comment:
Isn't there a type for images in `react`?

Reply:
Not any that I'm aware of.

Reply:
There is a type `ImageResolvedAssetSource`, but it also contains `scale` as a obligatory parameter, so I guess there aren't any indeed.